pkg/cmd/deprecation/deprecated.go

```diff
@@ -12,57 +12,17 @@ import (
 var deprecatedCommands = map[string]deprecationInfo{
 	"install": {
 		replacement: "jx boot",
-		date:        "Jun 1 2020",
+		date:        "Sep 1 2020",
 		info: fmt.Sprintf("Please check %s for more details.",
 			util.ColorStatus("https://jenkins-x.io/docs/getting-started/setup/boot/")),
 	},
+	// Todo: Implementation is very tightly coupled with jx install, better to remove it once we remove jx install
 	"init": {
 		replacement: "jx boot",
-		date:        "Jun 1 2020",
+		date:        "Sep 1 2020",
 		info: fmt.Sprintf("Please check %s for more details.",
 			util.ColorStatus("https://jenkins-x.io/docs/getting-started/setup/boot/")),
 	},
-	"create post": {
-		date: "Feb 1 2020",
-	},
-	"create spring": {
-		replacement: "jx create project",
-		date:        "Mar 1 2020",
-		info: fmt.Sprintf("Please check %s for more details.",
-			util.ColorStatus("https://jenkins-x.io/commands/jx_create_project/")),
-	},
-	"upgrade platform": {
-		replacement: "jx upgrade boot",
-		date:        "Jun 1 2020",
-		info: fmt.Sprintf("Please check %s for more details.",
-			util.ColorStatus("https://jenkins-x.io/docs/getting-started/setup/boot/")),
-	},
-	"upgrade ingress": {
```

Comment:
There are some occurrences in the jx codebase, is it safe to remove? I have marked a few places where it is used, what do people think about removing it?

Reply:
I think we might need it while we still use expose controller, something we try to get away from as well. 
@daveconde - does that sounds right?